sw/vendor/patches/riscv_compliance/0006-Fix-negative-immediate-in-assembly.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/riscv-test-suite/rv32i/src/I-RF_x0-01.S b/riscv-test-suite/rv32i/src/I-RF_x0-01.S
+index 8f282d502..c21d9d1a8 100644
+--- a/riscv-test-suite/rv32i/src/I-RF_x0-01.S
++++ b/riscv-test-suite/rv32i/src/I-RF_x0-01.S
+@@ -54,7 +54,7 @@ RV_COMPLIANCE_CODE_BEGIN
+     addi    x0, x0, 1
+     ori     x0, x0, 0x7F0
+     andi    x0, x0, 0x53F
+-    xori    x0, x0, 0xFFFFF803
++    xori    x0, x0, -0x7FD
```

Comment:
This seems fine for now but if there's a fix upstream can't we just update the entire compliance suite?

Reply:
I marked this as a draft in order to do that and I'll close this if I can get that to work.